packages/core/src/shared/featureConfig.ts

```diff
@@ -193,4 +193,28 @@ export class FeatureConfigProvider {
     public static getFeatureConfigs(): Map<string, FeatureContext> {
         return FeatureConfigProvider.instance.featureConfigs
     }
+
+    /**
+     * Retrieves the FeatureContext object for a given feature name.
+     *
+     * @param {string} featureName - The name of the feature.
+     * @returns {FeatureContext | undefined} The FeatureContext object for the specified feature, or undefined if the feature doesn't exist.
+     */
+    public static getFeature(featureName: string): FeatureContext | undefined {
+        return FeatureConfigProvider.instance.featureConfigs.get(featureName)
+    }
+
+    /**
+     * Checks if a feature is active or not.
+     *
+     * @param {string} featureName - The name of the feature to check.
+     * @returns {boolean} False if the variation is not CONTROL, otherwise True
+     */
+    public static isEnabled(featureName: string): boolean {
```

Comment:
can we more strongly type the featurename? Like have a list of constants somewhere?

Reply:
We don't have a list of feature flags or constant of such sort. I also believe such a constant can easily leak out unintended information.

Relevant Test file -
https://github.com/aws/aws-toolkit-vscode/blob/cd46a5a52ac57d2465ce8b1ec6103e82ef228295/packages/core/src/test/shared/featureConfig.test.ts#L94-L96

Reply:
What information could leak out? Any code that would call this is open source

Reply:
Done